lib/libpax/libpax_api.cpp

```diff
@@ -157,11 +157,21 @@ int libpax_counter_init(void (*init_callback)(void),
   return 0;
 }
 
+#define LIBPAX_STARTED 1
+#define LIBPAX_STOPPED -1
+int libpax_state = LIBPAX_STOPPED;
+
 int libpax_counter_start() {
   if (config_set == 0) {
     ESP_LOGE("configuration", "Configuration was not yet set.");
```

Comment:
```suggestion
    ESP_LOGE("libpax", "Configuration was not yet set, aborting libpax_counter_start.");
```

we should have a consistent TAG, i think we should just use `libpax` for all logs.

Reply:
This was resolved in the code.